packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -241,6 +241,111 @@ abstract class Action<T extends Intent> with Diagnosticable {
 /// If you listen to an [Action] widget in a widget hierarchy, you should use
 /// this widget. If you are using an [Action] outside of a widget context, then
 /// you must call removeListener yourself.
+///
+/// {@tool dartpad --template=stateful_widget_material}
+/// This example shows how ActionListener handles adding and removing of
+/// the [listener] in the widget lifecycle.
+///
+/// ```dart imports
+/// import 'package:flutter/material.dart';
+/// ```
+///
+/// ```dart preamble
+/// class ExampleActionListener extends StatefulWidget {
+///   @override
+///   _ExampleActionListenerState createState() => _ExampleActionListenerState();
+/// }
+///
+/// class _ExampleActionListenerState extends State<ExampleActionListener> {
+///   bool _on = false;
+///   MyAction _myAction;
+///
+///   @override
+///   void initState() {
+///     super.initState();
+///     _myAction = MyAction();
+///   }
+///
+///   @override
+///   Widget build(BuildContext context) {
+///     return Row(
```

Comment:
I'm not 100% sure, but you might need to wrap this in a Center widget.  When I tried running this locally, it aligned itself to the top when there were two children. It depends on what `--template=stateful_widget_material` does I guess.

<img width="436" alt="Screen Shot 2020-11-06 at 9 05 42 AM" src="https://user-images.githubusercontent.com/389558/98394292-5e158900-200f-11eb-9a50-1e2c3893b712.png">

Reply:
It seems that you were missing the scaffold on 338 - 345. It has a Center there.

Reply:
Ah ok thanks, this is fine as-is then 👍